lib/ts/recipe/session/types.ts

```diff
@@ -241,11 +249,17 @@ export type RecipeInterface = {
         userContext: UserContext;
     }): Promise<SessionContainerInterface | undefined>;
 
```

Comment:
in getSession, if we are doing checkDataabase: true, it should check for the user id / ip  ban as well. It can throw unauthoized error, just like refresh

Reply:
Also consider that we can utilise the cache for this?

Reply:
Also session operations like regenerateSession. You can check for user banning and ip here, cause we have access to the request object. This way, we also do these checks for email verification